react-client/src/Components/DomInfoDialogBox.js

```diff
@@ -1,6 +1,6 @@
 import React, { useeffect, useState } from "react";
 
-const DomInfoDialogBox = ({ id,idx,clstag,clsname,parenttag,parentId,parentClass,children,top, left, onClose,fontsize, fontfamily,textcolor
+const DomInfoDialogBox = ({ id,idx,clstag, clsname,parenttag,parentId,parentClass,children,top, left, onClose,fontsize, fontfamily,textcolor
```

Comment:
Be consistent with variable spacing please.

Reply:
They're inconsistent.

Reply:
Always use space after comma like so: `id, idx, clstag`

Reply:
sorry will correct had to edit during testing